server.js

```diff
@@ -0,0 +1,41 @@
+const express = require('express');
+require('dotenv').config();
+const app = new express();
+const port = process.env.PORT || 5000;
+
+app.use(express.static('public'));
```

Comment:
Same here; ensure `Cache-Control` header is sent for static files as well, current site is also using 5min TTL, so

```js
res.setHeader('Cache-Control', 'public, max-age=300')
```

Reply:
Resolved via 98116e5